src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -124,3 +128,18 @@ def test_delete_failure_not_found(resource_client, current_resource_model):
         Action.DELETE, OperationStatus.FAILED, current_resource_model
     )
     return error_code
+
+
+def test_input_equals_output(resource_client, input_model, output_model):
```

Comment:
why we remove the prune_null method?

Reply:
As discussed offline, if a user enters null/"" for a property we should check that against the output. Therfore, we don't need the prune_null method

Reply:
OK, I totally missed that. But I think null and empty still valuable so keep them is right way.